Library/Homebrew/download_strategy.rb

```diff
@@ -220,6 +229,10 @@ def extract_ref(specs)
   end
 end
 
+# The AbstractFileDownloadStrategy is a class necessary because using local
+# bottles instead of dowloading a file needs a few different commands. Also
+# there are two subclasses of the current class, CurlDownloadStrategy and
+# LocalBottleDownloadStrategy
```

Comment:
How about just `The AbstractFileDownloadStrategy is a class necessary because extracting local bottles rather than downloading a file needs different commands.`. Don't want to name specific classes or behaviour as the documentation will become outdated if the code changes.